commons-test/src/main/java/org/infinispan/commons/test/TestNGNameTranslator.java

```diff
@@ -0,0 +1,80 @@
+package org.infinispan.commons.test;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+
+import org.testng.ITestResult;
+import org.testng.annotations.Test;
+
+/**
+ * When a test is annotated with {@code @Test(testName = "foo")}, TestNG reports all the test methods in the class as
```

Comment:
```suggestion
 * When a test is annotated with {@code @Test(testName = "foo")}, {@code ITestResult.getTestName()} reports all the test methods in the class as
```

"reports" doesn't work very well here because it's no longer just fixing the XML reports.